template/en/default/mfa/page_requires_mfa.html.tmpl

```diff
@@ -0,0 +1,11 @@
+[% PROCESS "global/field-descs.none.tmpl" %]
+
+[% PROCESS global/header.html.tmpl
+   title  = "This page requires 2FA",
+   style_urls = ['skins/standard/page.css']         
+%]
+
+<p>Because you are a member of [% mfa_group FILTER html %] you must have 2FA enabled.
```

Comment:
See prior note about $mfa_group being an array someday.

Reply:
this hasn't been addressed - displaying the the `mfa_group` name here doesn't provide any information to the user.  change it to either list the groups that are direct members of `mfa_group`, or make the wording generic - "You are a member of a Bugzilla group that requires you to enable 2FA."